### DISCUSSION TITLE
dark-mode无法生效

### DISCUSSION THREAD

**Discussion by swing0828** [tw93/Pake#788] [Q&A]
在dark-mode无法生效，是 pake https://…… --name …… --dark-mode 这个命令不支持还是pa ke

**Comment by swing0828** [comment]
pake的URL本身不支持dark模式，比如闲鱼和豆瓣？

**Reply by tw93** [reply]
取决于站点本身是不是支持黑暗模式，你可以试试微信读书的场景，或者微信公众号页面去测试一下
